cnd/src/swap_protocols/rfc003/actions/ether.rs

```diff
@@ -17,7 +17,15 @@ impl FundAction<Ethereum, asset::Ether> for (Ethereum, asset::Ether) {
     type FundActionOutput = DeployContract;
 
     fn fund_action(htlc_params: HtlcParams<Ethereum, asset::Ether>) -> Self::FundActionOutput {
-        htlc_params.into()
+        let htlc = EtherHtlc::from(htlc_params.clone());
+        let gas_limit = htlc.deployment_gas_limit();
```

Comment:
```suggestion
        let gas_limit = htlc.fund_tx_gas_limit();
```

Reply:
📣 I think that's supposed to be the one for fund.

Reply:
Ha-ha. Good point. Well actually for Ether there is only one action and it is simply incorrectly named on Blockchain contracts side so I can't do the change here.

Reply:
Also note the code is moved so doing a request change on that is a bit harsh.